packages/taquito/test/estimate/rpc-estimate-provider.spec.ts

```diff
@@ -26,7 +26,9 @@ import {
 } from '../contract/helper';
 import { OpKind } from '@taquito/rpc';
 import { TransferTicketParams } from '../../src/operations/types';
-import { InvalidAddressError, InvalidContractAddressError } from '@taquito/utils';
+import { InvalidContractAddressError } from '@taquito/core';
```

Comment:
if line 29 and 30 are both importing from @taquito/core, will we consider to make it one line?